### PR TITLE
Fix Mirror

### PR DESCRIPTION
**Pull request by willronchetti**
- No longer rely on Torb to set an environment variable via EB
- Get this information directly from settings/whodaman
- XXX: tests?

### CODE REVIEW

File: src/encoded/__init__.py [hunk 1]
```diff
@@ -32,13 +32,29 @@
     json_asset
 )
 from dcicutils.log_utils import set_logging
+from dcicutils.beanstalk_utils import whodaman
```
Comment (netsettler) [inline src/encoded/__init__.py]: Although used internally here, this amounts to an export of this name from `encoded`. Minimally, I would do this import using `as _whodaman` to emphasize the intent is not that. (You'd have to change a usage farther down, too.)

We really should fix that naming at some point to something respectable.

File: src/encoded/__init__.py [hunk 2]
```diff
@@ -32,13 +32,29 @@
...
+    BEANSTALK_PROD_ENVS,
+)
 import structlog
 import logging
 
 # location of environment variables on elasticbeanstalk
 BEANSTALK_ENV_PATH = "/opt/python/current/env"
 
 
+def get_mirror_env(settings):
```
Comment (netsettler) [inline src/encoded/__init__.py]: I feel like there are lots of odd assumptions built into this, ending up feeling weirdly ad hoc, but at least it's a small function and the logic is plain. Maybe a little more documentation and/or just comments in the code explaining the presumptions would fix that. But it's not urgent.

File: src/encoded/__init__.py [hunk 3]
```diff
@@ -147,58 +163,6 @@ def app_version(config):
             config.registry.settings["ga_config"] = json.load(json_file)
 
 
-'''
```
Comment (netsettler) [inline src/encoded/__init__.py]: So this just wasn't being used? Huh.

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4910980&bid=16256672)

File: src/encoded/__init__.py [hunk 4]
```diff
@@ -29,16 +29,37 @@
...
+)
 import structlog
 import logging
 
 # location of environment variables on elasticbeanstalk
 BEANSTALK_ENV_PATH = "/opt/python/current/env"
 
 
+def get_mirror_env(settings):
+    """
```
Comment (j1z0) [inline src/encoded/__init__.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4910980)

File: src/encoded/__init__.py [hunk 5]
```diff
@@ -29,16 +29,37 @@
...
 import logging
 
 # location of environment variables on elasticbeanstalk
 BEANSTALK_ENV_PATH = "/opt/python/current/env"
 
 
+def get_mirror_env(settings):
+    """ 
+        Gets the mirror_env from whodaman instead of env variable 
+        This is important in our production environment because in our
```
Comment (j1z0) [inline src/encoded/__init__.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4910980)

File: src/encoded/__init__.py [hunk 6]
```diff
@@ -29,16 +29,37 @@
...
 import structlog
 import logging
 
 # location of environment variables on elasticbeanstalk
 BEANSTALK_ENV_PATH = "/opt/python/current/env"
 
 
+def get_mirror_env(settings):
+    """ 
+        Gets the mirror_env from whodaman instead of env variable
```
Comment (j1z0) [inline src/encoded/__init__.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4910980)

**Pull request merged**